tox.ini

```diff
@@ -92,6 +92,36 @@ skip_install = true
 allowlist_externals =
   git
 
+
+[testenv:preview-docs]
+description = preview the docs
+deps =
+  sphinx-autobuild
+  {[testenv:build-docs]deps}
+commands_pre =
+commands =
+  # Retrieve possibly missing commits:
+  -git fetch --unshallow
+  -git fetch --tags
+
+  # Build the html docs with sphinx-autobuild:
+  sphinx-autobuild \
```

Comment:
Can this be invoked via the runpy interface? It'd be nice to align it with `{envpython} -m` that the rest of the places use.

Reply:
> Can this be invoked via the runpy interface?

I tried that first, but `runpy` can't find `sphinx-autobuild`. I think that's because it's not installed via _requirements.txt_. Can I add it there?

Reply:
No, that's unrelated. `python -m` only accepts importables. Importables can't have dashes — they must be valid Python identifiers. What you wrote is a script name, the filename on disk that is created as an entry point.

Reply:
Looking into https://github.com/executablebooks/sphinx-autobuild/tree/main/src, I see that `sphinx_autobuild` is the importable we're looking for — this is what's installed into `site-packages/`. We can also confirm that it contains a `__main__.py` which makes it an entry point for runpy: https://github.com/executablebooks/sphinx-autobuild/tree/main/src/sphinx_autobuild.
If it didn't contain, `__main__.py`, the invocation would be `python -m sphinx_autobuild.<module_name>` but with that magic module, the invocation is simpler — it's `python -m sphinx_autobuild`.

Reply:
```suggestion
  {envpython} -m sphinx_autobuild \
```

Reply:
Thank you for explaining, researching and taking time to review in general!